src/components/feedbackFaces.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+import { event } from "@site/src/utils/gtags.client";
+
+// Default export function component
+export default function FeedbackFaces() {
+    const [clickedFace, setClickedFace] = useState<string | null>(null);
+
+    const faces = [
+        { label: "Sad", emoji: "😞" },
+        { label: "Neutral", emoji: "😐" },
+        { label: "Happy", emoji: "😊" },
+    ];
+
+    const handleFeedbackClick = (feedbackType: string) => {
+        setClickedFace(feedbackType);
+
+        const pagePath = window.location.pathname;
+        const label = `${feedbackType} - ${pagePath}`;
+
+        event({
+            action: "feedback_click",
+            category: "Feedback",
+            label: label,
+            location: pagePath,
+        });
+
+        // Reset the clicked state after a brief delay
+        setTimeout(() => setClickedFace(null), 300);
```

Comment:
Why do you want to let someone click more than one time? Do you just want to have in state its been clicked and disable clicking any further? I'd probably store something in local storage so the same person on the same browser can't click again imo.